### PR TITLE
Fuzzy def rule: prevent common false positives

### LINKED ISSUE #162: Make fuzzy def rule smarter
https://github.com/weavejester/cljfmt/blob/a4bf76174b8c43d18abcedec9072ef42f27cd423/cljfmt/resources/cljfmt/indents/fuzzy.clj

`"^def"` will match things like `default-translation-function` which aren't "defs", ruining indentation.

I propose placing a small blacklist of widely used words starting with `def` which aren't definitions, e.g. `[default defer deflate]`.

They could either be inlined in the existing regex, or expressed as code in core.clj.

Agreed?

Can contribute a PR.

Cheers - Victor

### PR DESCRIPTION
**Pull request by vemv**
Fixes #162

### CODE REVIEW

**Comment by vemv** [comment]
Had made a typo in the commit message. Ready now

**Comment by weavejester** [comment]
Thanks. Can you adjust the commit message as outlined in the [contributing guidelines](https://github.com/weavejester/cljfmt/blob/master/CONTRIBUTING.md)?

File: cljfmt/resources/cljfmt/indents/fuzzy.clj [hunk 1]
```diff
@@ -1,2 +1,2 @@
-{#"^def"   [[:inner 0]]
- #"^with-" [[:inner 0]]}
+{#"^def(?!ault)(?!late)(?!er)"   [[:inner 0]]
```
Comment (weavejester) [inline cljfmt/resources/cljfmt/indents/fuzzy.clj]: You can get rid of the extra spaces between `er)"` and `[[:in`

Comment (vemv) [reply cljfmt/resources/cljfmt/indents/fuzzy.clj]: Good catch! Amended

**Comment by vemv** [comment]
> Thanks. Can you adjust the commit message as outlined in the contributing guidelines?

Hey, sorry, hadn't noticed this message.

The commit message is 46 chars long, in imperative and with subject/body separation, so I fail to see the issue.

**Comment by vemv** [comment]
(Ready)

**Pull request merged**